File: 2st/Yeonwoo/가운데 글자 가져오기.py
def solution(s):
    answer = ''
    if len(s)%2==1:
        answer += s[len(s)//2]
    elif len(s)%2==0:
        answer += s[(len(s)//2)-1]
        answer += s[len(s)//2]
    return answer

File: 4st/Jisoo/programmer/가운데 글자 가져오기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        int len = s.length();
        if (len % 2 == 1) {
            return s.substring(len / 2, len / 2 + 1);
        } else {
            return s.substring(len / 2 - 1, len / 2 + 1);
        }
    }
}